src/type_conversion/hiwire.c

```diff
@@ -39,16 +33,20 @@ hiwire_bool(bool boolean)
 }
 
 EM_JS(int, hiwire_init, (), {
-  let _hiwire = { objects : new Map(), counter : 1 };
+  let _hiwire = {
+    objects : new Map(),
+    // Use a native uint32 for our counter. Counter starts at 1.
+    counter : new Uint32Array([1])
```

Comment:
This needs more extensive comments, including on overflow behavior and what odd and even values mean.